apps/database/lib/database/schema/mod_file.ex

```diff
@@ -12,4 +13,9 @@ defmodule Database.Schema.ModFile do
     many_to_many(:requires, Database.Schema.ModFile, join_through: "mod_dependencies")
     many_to_many(:required_by, Database.Schema.ModFile, join_through: "mod_dependencies")
   end
+
+  def changeset(record, params \\ :empty) do
```

Comment:
same dialyzer problem as with mod.ex here.
________________________________________________________________________________
apps/database/lib/database/schema/mod_file.ex:17:no_return
Function changeset/1 has no local return.
________________________________________________________________________________
apps/database/lib/database/schema/mod_file.ex:17:call
The call:
Database.Schema.ModFile.changeset(_any(), :empty)

will never return since it differs in arguments with
positions 2 from the success typing arguments:

(
  {map(), map()}
  | %{:__struct__ => atom() | %{:__changeset__ => map(), _ => _}, atom() => _},
  :invalid | %{:__struct__ => none(), (atom() | binary()) => _}
)
________________________________________________________________________________